chrome_manifest_v3/class_app.js

```
@@ -0,0 +1,848 @@
+  replacer(text, dict) {
+    let self = this;
+
+    if (!text || text.length < 1) {
+      // g2t_log('Require text!');
+      return '';
+    } else if (!dict || dict.length < 2) {
+      g2t_log('replacer: Require dictionary!');
+      return text;
+    }
+
+    let re, new_text;
```

Biome (1.9.4)

[error] Declare variables separately

Unsafe fix: Break out into multiple declarations


(lint/style/useSingleVarDeclarator)